src/wrench/services/compute/hadoop/DeterministicMRJob.cpp

```diff
@@ -8,13 +8,66 @@
  */
 
 #include "wrench/services/compute/hadoop/DeterministicMRJob.h"
+#include "wrench/services/compute/hadoop/hadoop_subsystem/Mapper.h"
 #include "wrench/logging/TerminalOutput.h"
 
 WRENCH_LOG_NEW_DEFAULT_CATEGORY(hadoop_compute_servivce, "Log category for Deterministic MR Job");
 
 namespace wrench {
 
-    DeterministicMRJob::DeterministicMRJob() : MRJob() {
+    DeterministicMRJob::DeterministicMRJob(int num_mappers, int num_reducers, double data_size,
+                                           bool use_combiner, int sort_factor, double spill_percent,
+                                           int key_value_width) : MRJob() {
+        this->setNumMappers(num_mappers);
+        this->setNumReducers(num_reducers);
+        this->setUseCombiner(use_combiner);
+        this->setDataSize(data_size);
+        this->setSortFactor(sort_factor);
+        this->setSpillPercent(spill_percent);
+        this->setKeyValueWidth(key_value_width);
+        this->setJobType(std::string("deterministic"));
+    }
+
+    // Typical set of parameters
```

Comment:
Don't know if this is necessary, but thought it might be useful for testing.

Reply:
sure, that can never hurt